createdisk.sh

```diff
@@ -86,7 +86,7 @@ kernel_release=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'uname -r')
 kernel_cmd_line=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'cat /proc/cmdline')
 
 # SCP the vmlinuz/initramfs from VM to Host in provided folder.
-${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/${BASE_OS}-${ostree_hash}/* $1
+${SCP} -r core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/${BASE_OS}-${ostree_hash}/* $1
```

Comment:
The commands from `ostree_hash=` to here could actually be wrapped in a `if [ -n "${SNC_GENERATE_MACOS_BUNDLE}" ]` as it's not going to be useful on linux/Windows.
I don't think hyperkit (which crc uses on macOS) works on M1 macs, so it might be a bit early to generate macOS  bundles by default in snc.
Could you at least expand the commit log a bit, is this causing an error if we omit the `-r`? Which additional files do we need from `/boot/ostree` which are in subdirs?

Reply:
so for aarch64 the kernel pulls in DTB - https://src.fedoraproject.org/rpms/kernel/blob/f34/f/kernel.spec#_1533 as part of the bootloading process which is the extra directory that requires the -r. 

Yes, so looks like hyperkit is not yet available for arm64 so I would just want to get these changes in for now so that in the future when it is available we can start on the CRC side of things.

Reply:
changed to have this block under the conditional.